apps/frontend/pages/404.js

```diff
@@ -0,0 +1,35 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.papiez}>
+
+      <h1 className={styles.one}><b>404.</b></h1>
+
+      <a className={styles.txt2}>Może szukałeś czegoś innego? Sprawdź to:</a>
+
+      <br></br>
+      <br></br>
+      <div className={styles.divlist}>
+
+      <ul className={styles.listmenu}>
+        <li>
+          <a className={styles.mainref} href="/">Strona Główna</a>
+        </li>
+        <br></br>
```

Comment:
don't use `<br>` for spacing - use css instead

to do it easily, you can add `display: grid` to `<ul>` with `gap: ...px`